t/plugin/openid-connect.t

```diff
@@ -112,7 +112,7 @@ done
                         },
                         "upstream": {
                             "nodes": {
-                                "127.0.0.1:1980": 1
+                                "127.0.0.1:8888": 1
```

Comment:
`8888`, Is it a valid "openid" port?

Reply:
Hey, I'm trying to add tests to check if the plugin sets the headers correctly as configured. I don't have a local development setup yet, so have to run the checks on this PR on GitHub to see if it's working.

I switched the upstream port to `8888` since that's where a `mendhak/http-https-echo` container is running. This one returns in the response body the full content of the incoming request as a JSON string. So I think I can check if the plugin has set the headers correctly by checking the response body. If the tests still work with the new port, I will try to add a case to do some actual verification of the headers.

Reply:
@jenskeiner 
You can setup the test environment via https://github.com/apache/apisix/blob/master/.travis/linux_openresty_runner.sh

Reply:
Yes, thanks for the link. It's more that I have to set up a fresh Linux VM or similar since I want that separate from the rest.

Reply:
And you can write your echo-back here: https://github.com/apache/apisix/blob/master/t/lib/server.lua

Reply:
Understood, seems to be the better option. Will adjust the code.